CMakeModules/EosioTester.cmake.in

```diff
@@ -8,6 +8,15 @@ set(EOSIO_VERSION "@VERSION_MAJOR@.@VERSION_MINOR@.@VERSION_PATCH@")
 
 enable_testing()
 
+if (UNIX)
+   if (APPLE)
+      if (LLVM_DIR STREQUAL "" OR NOT LLVM_DIR)
+         set(LLVM_DIR "/usr/local/Cellar/llvm@4/4.0.1/lib/cmake/llvm")
```

Comment:
Why not have the path be `/usr/local/opt/llvm@4/lib/cmake/llvm`? That way you don't have a dependency on the particular minor and patch release of LLVM? And that symlink should exist even if the user `brew unlink`s `llvm@4`.

Same goes for the path in EosioTesterBuild.cmake.in.